src/app/messages/es/messages.json

```diff
@@ -125,6 +125,7 @@
   "i18n:notification:toast:highlights:popup:load-failure": "No se pudo cargar algunos textos resaltados. Por favor, compruebe su conexión y vuelva a intentarlo.",
   "i18n:notification:toast:highlights:popup:print-failure": "No podemos imprimir esta página. Por favor, compruebe su conexión y vuelva a intentarlo.",
   "i18n:notification:toast:study-guides:load-failure": "No podemos cargar las guías de estudio en esta página. Por favor, compruebe su conexión y vuelva a intentarlo.",
+  "i18n:notification:toast:highlights:select-overlap": "No se admiten resaltados superpuestos.",
```

Comment:
@bethshook Can you check this translation, please?

Reply:
looks good